### ISSUE TITLE
Prioritize video from session on map data

### ISSUE THREAD

**Issue by flyingzumwalt** [ipfs/2016-Q3-Workshop#22]
Sounds like the session wasn't streamed or recorded, in which case this is a moot point. If it _was_ recorded, please prioritize posting that video. It will be useful to review and to recommend for others to watch.

**Comment by RichardLitt** [comment]
I would happily review this, if it was recorded.

**Comment by daviddias** [comment]
It was recorded, but unfortunately not with the best set up, which requires me to process the video (slice, attach the sound from the mic, stitch together parts because the battery run out and so on).

I got more than 100GB of video to tag, I'll need a couple of weeks until I can get back to that.

**Comment by em-ly** [comment]
@diasdavid do you have to be the one to process it or can we outsource that to expedite the process?

**Comment by daviddias** [comment]
Once I've it tagged by days+discussions, then we can outsource the editing.

**Comment by flyingzumwalt** [comment]
FYI I have a friend in MN who's looking for video editing work right now, though I'm sure you already have video editors who you work with.

**Comment by whyrusleeping** [comment]
I dont think we actually have a person for video editing. AFAIK its just been @diasdavid being super awesome and doing it
